frontend/app/routes/about.js

```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  model() {
+    return 'badgeyayofficial@gmail.com';
```

Comment:
What is the use case when you statically type the email here instead of fetching it from the DB or from the frontend through local storage?